src/Model/Item.php

```diff
@@ -2122,8 +2122,10 @@ private static function getLanguage(array $item): ?string
 	 * @param array   $default
 	 * @return array
 	 */
-	public static function getLanguageArray(string $body, int $count, int $uri_id = 0, int $author_id = 0, array $default = ['un' => 1]): array
+	public static function getLanguageArray(string $body, int $count, int $uri_id = 0, int $author_id = 0, array $default = []): array
 	{
+		$default = $default ?: ['un' => 1];
```

Comment:
Please make the `'un'` string into a constant with a meaningful name, like `UNDETERMINED_LANGUAGE`.

Reply:
Done